ironfish-rust-nodejs/index.ts

```diff
@@ -158,6 +158,10 @@ export class Transaction {
 
     return new TransactionPosted(native.transactionPost.call(this.boxedData, spenderHexKey, changeGoesTo, intendedTransactionFee.toString()));
   }
+
+  set_expiration_sequence(expirationSequence: number): undefined {
```

Comment:
I know we're not very consistent about this (e.g. post_miners_fee), but let's camel-case this (e.g. setExpirationSequence) so it falls in line with typical JS function names.

Reply:
Updated